recipe/meta.yaml

```diff
@@ -19,11 +19,11 @@ requirements:
   host:
     - pip
     - python >=3.6
-    - setuptools_scm
+    - setuptools-scm
   run:
     - attrs >=17.4.0
     # TODO: upstream, this is conditional to <3.8, but probably not worth the non-noarch build
-    - importlib_metadata
+    - importlib-metadata
```

Comment:
This does not change anything in the package really. The `_` packages there aliases to the `-`.
That is you motivation for this change?

Reply:
No, functionality of the package is not changed. But if you export/pin environments you will obtain duplicated packages. 
```
...
  - importlib-metadata=2.0.0=py_1
  - importlib_metadata=2.0.0=1
...
```
What is the benefit of using the alias, which was the former package name? It is very helpful package name transitions indeed.